depth_image_proc/CMakeLists.txt

```diff
@@ -1,6 +1,8 @@
 cmake_minimum_required(VERSION 2.8)
 project(depth_image_proc)
 
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11")
```

Comment:
I actually cannot allow that during a release cycle. I'd love to but we'll have to wait until Kinetic.